### PR TITLE
method PUT for RequestBuilding

### PR DESCRIPTION
**Pull request by Nurmukhanov**
Look....

### CODE REVIEW

File: test/walkingdevs/req/FakeServerForPut.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package walkingdevs.req;
```
Comment (aibar) [inline test/walkingdevs/req/FakeServerForPut.java]: This class is unnecessary

File: test/walkingdevs/req/ReqTest.java [hunk 2]
```diff
@@ -3,16 +3,25 @@
...
-import walkingdevs.http11.RespBody;
+import walkingdevs.http11.*;
 import walkingdevs.str.Str;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 
 // TODO: more tests
 public class ReqTest extends Assert {
+
```
Comment (aibar) [inline test/walkingdevs/req/ReqTest.java]: again unnecessary

**Comment by Nurmukhanov** [comment]
fixed

File: test/walkingdevs/req/ReqTest.java (hunk 2 above)
Comment (Nurmukhanov) [inline test/walkingdevs/req/ReqTest.java]: Fixed

**Pull request merged**